tasks/src/image-to-image/data.ts

```diff
@@ -0,0 +1,60 @@
+import { type TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Synthetic dataset, simulating fog on real scenes",
+			id: "FoggyCityscapes"
+		},
+		{
+			description: "Synthetic dataset, for image relighting",
+			id: "VIDIT"
+		},
+		{
+			description: "A collection of selfie images and similar faces from anime"
+			id: "selfie2anime"
+		},
+		{
+			description: "Multiple images of celebrities, used for facial expression translation"
+			id: "CelebA"
+		}
+	],
+	demo:     {
+		inputs:  [],
+		outputs: [],
+	},
+	isPlaceholder: false,
+	metrics:       [
+		{
+			description: "Peak Signal to Noise Ratio (PSNR) is an approximation of the human perception, considering the ratio of the absolute intensity with respect to the variations. Measured in dB, a high value indicates a high fidelity.",
+			id:          "PSNR"
+		},
+		{
+			description: "Structural Similarity Index (SSIM) is a perceptual metric which compares the luminance, constrast and structure of two images. Between -1 and 1, the higer values indicate closer resemblance.",
+			id:          "SSIM"
+		},
+		{
+			description: "Inception Score (IS) is an analysis of the labels predicted by an image classification model when presented with a sample of the generated images.",
```

Comment:
How can we use inception score in this task?

Reply:
IS can be used (with ups and downs) to evaluate the output of generative models by analyzing the distribution of the output. Very intuitively, if we want to make image to image translation of a tree, we can run the results through a classification model to make sure we still have a tree, and asses the confidence of the classifier for the quality of the image.

https://arxiv.org/pdf/1801.01973.pdf
https://arxiv.org/pdf/2101.08629.pdf (section II c)